docs/src/gen/consumer/file.rst

```diff
@@ -42,18 +42,20 @@ Parameters
 **File**
 
   This value is a mandatory setting and contains the name of the
-  file to read. The file will be read from beginning to end and the reader
-  will stay attached until the consumer is stopped, so appends to the
-  file will be recognized automatically.
+  file to read. This filed supports glob patterns.
```

Comment:
Typo: `This field supports glob patterns.`

Reply:
fixed (line moved)